config/services.yaml

```diff
@@ -55,3 +53,7 @@ services:
     App\ValueObject\PdfOrientation:
         arguments:
             $orientation: '%pdf_orientation%'
+
+    App\ValueObject\Currency:
+        arguments:
+            $currency: '%currency%'
```

Comment:
Please use a syntax that is not involving the parameter name (either dashes or []). We should do an issue to change the one above ($orientation)